lib/src/query_component.dart

```diff
@@ -21,7 +21,7 @@ import 'util.dart';
   ],
   directives: [bsAccordionDirectives, BsInput, coreDirectives, formDirectives],
 )
-class QueryComponent implements OnInit {
```

Comment:
Loading the Office JS library on component init is also moved to root component, since it needs to know whether the app is sideloaded in Excel taskpane or not.